### ISSUE TITLE
Testing Advanced Search Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4635]
# Page to Test:

Advanced Search - https://openlibrary.org/advancedsearch

# Steps

Steps for this task are outlined in #4521
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly No images present on this page. We can close this issue.
